src/main.js

```diff
@@ -1,83 +1,83 @@
-//       "https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80",
-//     title: "Rhino Family",
-//     description: "Mother (or father) rhino with two babies",
-//     keyword: "rhino",
-//     horns: 2,
-//   },
-
-//   {
-//     image_url:
-//       "https://www.dhresource.com/0x0s/f2-albu-g5-M00-1A-11-rBVaI1hsIIiALxKzAAIHjSU3VkE490.jpg/wholesale-halloween-costume-prop-unicorn.jpg",
-//     title: "Unicorn Head",
-//     description: "Someone wearing a creepy unicorn head mask",
-//     keyword: "unicorn",
-//     horns: 1,
-//   },
-// ];
-
-// function displayBeasts(array) {
-//   for (let beast of array) {
-//     let img = this.image_url;
-//     let alt = "horned beast image";
-//     let title = this.title;
-//     let disc = this.description;
-
-//     return `<HornedBeasts imgSrc="${img}" alt="${alt} title="${title} />`
-//     console.log("beasts func is funcin")
-//   }
-// }
-
-// let singleVariable = displayBeasts(beastsArray);
-// console.log(singleVariable)
+import beastData from './data.json';
+import Image from "react-bootstrap/Image";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Container from "react-bootstrap/Container";
 
 class Main extends Component {
   
   render() {
     return (
-      <>
+      <Container>
         <h1>Main is maining</h1>
+        <Row>
+          <Col>
         <HornedBeasts
-          imgSrc="https://place-hold.it/300x500"
-          alt="Horned Image"
-          title="idk horned title"
+          imgSrc={beastData[0].image_url}
+          alt={beastData[0].keyword}
+          title={beastData[0].title}
+          description={beastData[0].description}
         />
+          </Col>
+          <Col>
         <HornedBeasts
-          imgSrc="https://place-hold.it/300x500"
-          alt="Horned Image"
-          title="idk horned title"
+          imgSrc={beastData[1].image_url}
+          alt={beastData[1].keyword}
+          title={beastData[1].title}
+          description={beastData[1].description}
         />
-      </>
+          </Col>
+          <Col>
+        <HornedBeasts
+          imgSrc={beastData[2].image_url}
+          alt={beastData[2].keyword}
+          title={beastData[2].title}
+          description={beastData[2].description}
+        />
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+        <HornedBeasts
+          imgSrc={beastData[3].image_url}
+          alt={beastData[3].keyword}
+          title={beastData[3].title}
+          description={beastData[3].description}
+        />
+          </Col>
+          <Col>
+          </Col>
+          <Col>
+          </Col>
+        </Row>
+      </Container>
     );
   }
 }
 
 class HornedBeasts extends Component {
 
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      favorites: 0
+    }
+  }
+
   render() {
     return (
       <>
```

Comment:
Hi there!

I was able to get your app running after removing your fragments (and replacing them with divs), and then checked all your imports of react component.

Please make sure you are importing components like this:

`import React, { Component } from 'react';` Looks like you were missing commas. 👍